### DISCUSSION TITLE
SSRIs + Opioids, Tramadol

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#90] [Drug Interactions]
SSRIs + Opioids

There have been some case reports of serotonin syndrome in individuals who were given a combination of opioids and SSRIs . Other research has indicated a small increase in overdose risk for people who are prescribed CYP2D6-inhibiting SSRIs (e.g. fluoxetine and paroxetine) with oxycodone. Overall, risk is low with this combination but, as always, be mindful of your consumption.

SSRIs + tramadol
Unsafe due to increased risk of serotonin toxicity.

Ansari, H., & Kouti, L. (2016). Drug interaction and serotonin toxicity with opioid use: Another reason to avoid opioids in headache and migraine treatment. Current Pain and Headache Reports, 20(8), 50. https://doi.org/10.1007/s11916-016-0579-3

Baldo, B. A. (2021). Toxicities of opioid analgesics: respiratory depression, histamine release, hemodynamic changes, hypersensitivity, serotonin toxicity. Archives of Toxicology, 95(8), 2627–2642. https://doi.org/10.1007/s00204-021-03068-2

Baldo, B. A., & Rose, M. A. (2020). The anaesthetist, opioid analgesic drugs, and serotonin toxicity: a mechanistic and clinical review. British Journal of Anaesthesia, 124(1), 44–62. https://doi.org/10.1016/j.bja.2019.08.010

Yunusa, I., Gagne, J. J., Yoshida, K., & Bykov, K. (2022). Risk of opioid overdose associated with concomitant use of Oxycodone and selective serotonin reuptake inhibitors. JAMA Network Open, 5(2), e220194. https://doi.org/10.1001/jamanetworkopen.2022.0194

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/315 and https://github.com/TripSit/drugs/issues/316
